kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -353,6 +403,32 @@ public void testManipulateProcessVariable() throws Exception {
         }
 
     }
+    
+    public void testManipulateProcessVariableWrongContainer() throws Exception {
```

Comment:
Checked exceptions are never thrown within this test
```suggestion
    public void testManipulateProcessVariableWrongContainer() {
```